src/parameterizations/lateral/MOM_thickness_diffuse.F90

```diff
@@ -1969,6 +1989,22 @@ subroutine thickness_diffuse_init(Time, G, GV, US, param_file, diag, CDp, CS)
   call get_param(param_file, mdl, "KHTH", CS%Khth, &
                  "The background horizontal thickness diffusivity.", &
                  default=0.0, units="m2 s-1", scale=US%m_to_L**2*US%T_to_s)
+  call get_param(param_file, mdl, "READ_KHTH", CS%read_khth, &
+                 "If true, read a file (given by KHTH_FILE) containing the "//&
+                 "spatially varying horizontal thickness diffusivity.", default=.false.)
+  if (CS%read_khth) then
+    if (CS%Khth > 0) then
+        call MOM_error(FATAL, "thickness_diffuse_init: KHTH > 0 is not "// &
+              "compatible with READ_KHTH = TRUE. ")
+    endif
+    call get_param(param_file, mdl, "KHTH_FILE", khth_file, &
+                 "The file containing the spatially varying horizontal "//&
+                 "thickness diffusivity.", default="INPUT/khth.nc")
+
```

Comment:
In all of the other cases where we read a file name, we allow the directory to be set independently via the parameter INPUTDIR, and then log the full file path separately as what would be 'INPUTDIR/KHTH_FILE' in this case.  Doing so allows us to self-consistently specify the input directory for all MOM6 input with only a single line modification to a `MOM_input` or `MOM_override` file.  Search through the code for 'INPUTDIR' for many examples of this construct.  Please modify the code in this PR to follow this pattern.